x-pack/platform/plugins/shared/cases/server/plugin.ts

```diff
@@ -252,6 +262,9 @@ export class CasePlugin
             savedObjectsService: savedObjects,
           });
         },
+        scheduleIdCrementerTask: () => {
```

Comment:
```suggestion
        scheduleIncrementerTask: () => {
```
or `scheduleIdIncrementerTask`?